catalog/app/containers/Bucket/PackageDialog/PackageCreationForm.tsx

```diff
@@ -62,6 +66,66 @@ export interface PackageCreationSuccess {
   hash?: string
 }
 
+function createReadmeFile(name: string) {
+  const contents = [
+    `# ${name}`,
+    '\n\n',
+    `Stub README for the **${name}** package generated by Quilt Catalog`,
+    '\n',
+  ]
+  const f = new File(contents, README_PATH, { type: 'text/markdown' })
+  return FI.computeHash(f) as FI.LocalFile
+}
+
+// XXX: move to dialogs module
```

Comment:
Would be good if you mean this module app/components/Dialog/Confirm.tsx

Reply:
i mean `app/utils/Dialogs.js`